tests/int/test.test.ts

```diff
@@ -0,0 +1,23 @@
+import {createPacts} from '../../src/core/create-pacts';
```

Comment:
Rename test filename to something more descriptive :)

Reply:
renamed